pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -37,8 +38,8 @@ const (
 	bottlerocketUserData     = `
 [settings.kubernetes]
 api-server = "{{.Cluster.Endpoint}}"
-cluster-certificate = "{{.Cluster.CABundle}}"
-cluster-name = "{{.Cluster.Name}}"
+cluster-certificate = "{{if .Cluster.CABundle}}{{.Cluster.CABundle}}{{end}}"
```

Comment:
If there any chance that this will cause the code `cluster-certificate = ` which might be misinterpreted by bottlerocket? Is that valid TOML?

Does it make more sense to wrap the entire line like we do with labels?